armsrc/fpgaloader.h

```diff
@@ -57,6 +57,7 @@ void SetAdcMuxFor(uint32_t whichGpio);
 #define FPGA_LF_EDGE_DETECT_TOGGLE_MODE				(1<<1)
 // Options for the HF reader, tx to tag
 #define FPGA_HF_READER_TX_SHALLOW_MOD				(1<<0)
+#define FPGA_HF_READER_TX_FAST_SSP_CLK				(1<<1)
```

Comment:
Do we really need to make this an option? Why not just increase the clock rate?

Reply:
Withouf modifications: of course. Let's go with the option to simplify things...